src/components/domain/project/projectSearch/ProjectSearchBar.tsx

```diff
@@ -72,22 +70,26 @@ export default function ProjectSearchBar({ className }: ProjectSearchBarProps) {
         <div className="mt-2">
           {/* NOTE : 태그 회의 후 상세보기 내용 추가 예정 */}
           {/* #20240710.syjang, CheckTagInput 사용 예시 작성하려 임의로 코드 작성했습니다. 아래는 참고만 부탁드립니다. */}
-          <div className="flex gap-4">
-            <TagInput className="px-3 py-1 display6 tag-indigo" defaultValue="카테고리" />
-            <div className="flex gap-1">
-              {ProjectCategories.map((category) => {
-                return (
-                  <CheckTagInput
-                    key={category}
-                    value={category}
-                    isChecked={isCategorySelected(category)}
-                    isDisabled={isSelectionLimitReached() && !isCategorySelected(category)}
-                    onClick={() => {
-                      selectCategory(category);
-                    }}
-                  />
-                );
-              })}
+          <div className="flex flex-col gap-3">
+            <div className="flex flex-wrap gap-3">
+              <span className="h-9 w-20 rounded-[6px] bg-indigo-50 px-3 py-2 text-center text-indigo-500 display6">
+                카테고리
+              </span>
+              <div className="flex flex-wrap gap-1">
+                {ProjectCategories.map((category) => {
+                  return (
+                    <CheckTagInput
+                      key={category}
+                      value={category}
+                      isChecked={isSelected(category)}
+                      isDisabled={isSelectionLimitReached() && !isSelected(category)}
+                      onClick={() => {
```

Comment:
이 부분도 마찬가지로 jsx 외부 상단에 타입과 함께 함수를 선언하는게 어떨까요?
여긴 함수가 짧긴 하지만, 어떤 함수이고 어떤 타입인지 명확한 편이 좋다고 생각합니다!
```
  const handleCategoryClick = (category: string) => {
    addSelectList(category);
  };

```

```suggestion
                      onClick={() => handleCategoryClick(category)}
```

Reply:
✅ fae9e7c86557468476bb00d8dd9f3ad607a336ac